package.json

```diff
@@ -10,6 +11,7 @@
   "devDependencies": {
     "@types/jest": "^26.0.24",
     "jest": "^27.0.6",
+    "nodemon": "^2.0.12",
     "prettier": "^2.3.2",
     "ts-jest": "^27.0.4",
```

Comment:
I can see that it is specified in the dependencies. Is that not enough?

Reply:
`ts-node` is not specified in `package.json`, so it will not be installed when I run `npm install` in the project. Are you confusing it with `ts-jest`?

Reply:
I meant line 7 on the package.json

Reply:
Can you paste the line you mean here? I don't see anything like that on line 7.